docs/usage/tutorial.rst

```diff
@@ -739,99 +829,109 @@ population. In essence, a population is just refined rejection sampling.
-    Posterior means for population 1: t1: 0.559, t2: 0.23
-    Posterior means for population 2: t1: 0.552, t2: 0.205
+    Sample means for population 0: t1: 0.547, t2: 0.232
+    Sample means for population 1: t1: 0.556, t2: 0.236
+    Sample means for population 2: t1: 0.556, t2: 0.22
     
 
 
-.. code:: ipython3
+.. code:: python
 
-    result_smc.plot_marginals_all_populations(bins=25, figsize=(8, 2), fontsize=12)
+    result_smc.plot_marginals(all=True, bins=25, figsize=(8, 2), fontsize=12)
 
 
 
-.. image:: http://research.cs.aalto.fi/pml/software/elfi/docs/0.6/usage/tutorial_files/tutorial_81_0.png
+.. image:: http://research.cs.aalto.fi/pml/software/elfi/docs/0.6.1/usage/tutorial_files/tutorial_89_0.png
 
 
 
-.. image:: http://research.cs.aalto.fi/pml/software/elfi/docs/0.6/usage/tutorial_files/tutorial_81_1.png
+.. image:: http://research.cs.aalto.fi/pml/software/elfi/docs/0.6.1/usage/tutorial_files/tutorial_89_1.png
 
 
 
-.. image:: http://research.cs.aalto.fi/pml/software/elfi/docs/0.6/usage/tutorial_files/tutorial_81_2.png
+.. image:: http://research.cs.aalto.fi/pml/software/elfi/docs/0.6.1/usage/tutorial_files/tutorial_89_2.png
 
 
 Obviously one still has direct access to the samples as well, which
 allows custom plotting:
 
-.. code:: ipython3
+.. code:: python
 
     n_populations = len(schedule)
     fig, ax = plt.subplots(ncols=n_populations, sharex=True, sharey=True, figsize=(16,6))
-    samples = [pop.samples_list for pop in result_smc.populations]
-    for ii in range(n_populations):
-        s = samples[ii]
-        ax[ii].scatter(s[0], s[1], s=5, edgecolor='none');
-        ax[ii].set_title("Population {}".format(ii));
-        ax[ii].plot([0, 2, -2, 0], [-1, 1, 1, -1], 'b')
-    ax[0].set_xlabel(result_smc.names_list[0]);
-    ax[0].set_ylabel(result_smc.names_list[1]);
+    
+    for i, pop in enumerate(result_smc.populations):
+        s = pop.samples
+        ax[i].scatter(s['t1'], s['t2'], s=5, edgecolor='none');
+        ax[i].set_title("Population {}".format(i));
+        ax[i].plot([0, 2, -2, 0], [-1, 1, 1, -1], 'b')
+        ax[i].set_xlabel('t1');
+    ax[0].set_ylabel('t2');
     ax[0].set_xlim([-2, 2])
     ax[0].set_ylim([-1, 1]);
 
 
 
-.. image:: http://research.cs.aalto.fi/pml/software/elfi/docs/0.6/usage/tutorial_files/tutorial_83_0.png
+.. image:: http://research.cs.aalto.fi/pml/software/elfi/docs/0.6.1/usage/tutorial_files/tutorial_91_0.png
 
 
 It can be seen that the populations iteratively concentrate more and
 more around the true parameter values.
 
 Note that for the later populations some of the samples lie outside
 allowed region. This is due to the SMC algorithm sampling near previous
-samples, with *near* meaning a Gaussian distribution centered around
-previous samples with variance as twice the weighted empirical variance.
-However, the outliers carry zero weight, and have no effect on the
-estimates.
+samples. However, the outliers carry zero weight, and have no effect on
```

Comment:
However, the outliers carry zero weight and have no effect on...
(not an independent clause after "and" so no comma)